### ISSUE TITLE
Be able to configure the warning "Could not find suitable TypeScript type for X". 'any' assumed.

### ISSUE THREAD

**Issue by PandaWood** [reinforced/Reinforced.Typings#36]
It seems OK for us to rely on this default behavior so I would like to not see warnings for it.
Perhaps we could either 1) have a config item for suppressing all Typings WARNINGS or 2) particular warnings could be suppressed - in this case "RTW0003_TypeUnknown"

**Comment by PandaWood** [comment]
#37 Is a start

**Comment by pavel-b-novikov** [comment]
I will check soon

**Comment by PandaWood** [comment]
No problem, I will try to continue working on this - and confirm that it works - I just wanted to give you something to show that I'm on it and nearly there

**Comment by pavel-b-novikov** [comment]
I'd like to make fluent configuration that would allow to select multiple different messages to ignore instead of igonoring perticular message

**Comment by PandaWood** [comment]
Actually, I couldn't find in the documentation, a way to override the type or somehow suppress this warning - just using attributes
Do you know of a way to do this?
It's just that I have over 50 warnings from ReinforcedTypings I'd really like to remove

```
Severity	Code	Description	Project	File	Line	Suppression State
Warning		Could not find suitable TypeScript type for System.DateTime. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for System.DateTime. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for System.DateTime. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for System.DateTime. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for CC.Domain.Writeback.WritebackCatalogDTO. 'any' assumed.	CC.Core	C:\code\CC\src\CC.Core\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for CC.Domain.Writeback.WritebackCatalogDTO. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for CC.Domain.Subset. 'any' assumed.	CC.Core	C:\code\CC\src\CC.Core\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for CC.Domain.Subset. 'any' assumed.	CC.Web.Site	C:\code\CC\src\CC.Web.Site\Reinforced.Typings		
Warning		Could not find suitable TypeScript type for CC.Domain.Scheduler.Jobs.JobScheduleType. 'any' assumed.	CC.Core	
...
```

**Comment by pavel-b-novikov** [comment]
Dear Panda, the more flexible solution will be provided soon. :)

**Comment by PandaWood** [comment]
Sorry I didn't finish my Pull Request yet

**Comment by pavel-b-novikov** [comment]
Reopen?

**Comment by stevenfowler16** [comment]
Checking up on this. Looking to not have the warnings appear on the editor but show up in the build process.

**Comment by pavel-b-novikov** [comment]
Is it really critical?

**Comment by stevenfowler16** [comment]
Not critical but sometimes i'm choosing to not include the type on purpose. Such as for a database object that could have a decent amount of types but types I'm not going to use in my JavaScript to prevent file bloat.

**Comment by pavel-b-novikov** [comment]
Warning suppression mechanism delivered: `RtSuppress` in Reinforced.Typings.settings.xml

**Comment by buddde** [comment]
Hello, `<RtSuppress>RTW0003</RtSuppress>` does not work:

Reinforced.Typings : Type resolvation warning RT0003: Could not find suitable TypeScript type for X.Y.Z. 'any' assumed.

What do I have to change to suppress this warning?

**Comment by buddde** [comment]
Yes, i did install version 1.6.0

**Comment by pavel-b-novikov** [comment]
Fine. Yup, I see why. Will fix shortly

**Comment by pavel-b-novikov** [comment]
Just published 1.6.1. Issues is fixed there.

**Comment by buddde** [comment]
Thank you for your quick fix.
Warning suppression works great now in 1.6.1
